pkgs/applications/networking/cluster/habitat/default.nix

```
@@ -33,8 +31,7 @@ buildRustPackage rec {
     description = "An application automation framework";
     homepage = https://www.habitat.sh;
     license = licenses.asl20;
-    maintainers = [ maintainers.rushmorem ];
-    platforms = [ "x86_64-linux" "x86_64-darwin" ];
-    broken = true; # mark temporary as broken due git dependencies
+    maintainers = [ maintainers.boj maintainers.rushmorem ];
+    platforms = [ "x86_64-linux" ];
```

Why removing darwin support?
I just successfully build the updated package on MacOS with a minor change.

```suggestion
    platforms = [ "x86_64-linux" "x86_64-darwin" ];
```

---

It was broken last I tried and had no way to test it.

---

Understood. It should work with the proposed changes :)